static/game/bullet.js

```diff
@@ -62,18 +66,16 @@ export class Bullet extends Entity {
     this.collideMap.set(
       entityType,
       /** @param {import ("./creature.js").Creature} c */ c => {
-        if (entityType === "Enemy" || isCollidingCheat(c, this, CHEAT_RADIUS)) {
-          // deal basic damage
-          c.takeDamage(this.damage, this.vel);
-          // impart momentum
-          const size = (c.width * c.height) / 300;
-          c.vel = c.vel.add(this.vel.mult(this.knockback / size));
-          // call onHitEnemy functions
-          for (const ohe of this.onHitEnemy) {
-            if (ohe.func) ohe.func(this, ohe.data, c);
-          }
-          this.deleteMe = true;
+        // deal basic damage
+        c.takeDamage(this.damage);
```

Comment:
`Creature.takeDamage()` takes a second optional argument of the direction the damage came from now, for the purpose of converting overkill damage into velocity. Make this `c.takeDamage(this.damage, this.vel.norm2())`